cmd/stack_package.go

```
@@ -93,12 +100,34 @@ The packaging process builds the stack image, generates the "tar.gz" archive fil
   Packages the stack in the current directory, tags the built image with the default registry and namespace, and adds the stack to the "dev.local" repository.
   
   appsody stack package --image-namespace my-namespace
-  Packages the stack in the current directory, tags the built image with the default registry and "my-namespace" namespace, and adds the stack to the "dev.local" repository.`,
+  Packages the stack in the current directory, tags the built image with the default registry and "my-namespace" namespace, and adds the stack to the "dev.local" repository.
+  
+  appsody stack package --buildah --buildah-options "--format=docker"
+  Packages the stack in the current directory, builds project using buildah primitives in Docker format, tags the built image with the default registry and namespace, and adds the stack to the "dev.local" repository.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 
 			log.Info.Log("******************************************")
 			log.Info.Log("Running appsody stack package")
 			log.Info.Log("******************************************")
+			var err error
+			buildOptions := ""
+			if config.buildahBuildOptions != "" {
+				if !config.Buildah {
+					err = errors.New("Cannot specify --buildah-options flag without --buildah")
+				}
+				buildOptions = strings.TrimSpace(config.buildahBuildOptions)
+			}
+
+			if config.dockerBuildOptions != "" {
+				if config.Buildah {
+					err = errors.New("Cannot specify --docker-options flag with --buildah")
+				}
+				buildOptions = strings.TrimSpace(config.dockerBuildOptions)
+			}
+
+			if err != nil {
```

Why not return the errors as they are happening above?